common/source/docs/common-auxiliary-functions.rst

```diff
@@ -594,6 +596,7 @@ Other functions are:
     ICEngine start / stop                 Controls ICE engine starter output :ref:`ICE engines<common-ice>`
     Autotune_test_gains                   After autotune,allows selecting tune(high) or pretune gains
     VTOL QuickTune                        Autotunes QuadPlane VTOL PIDs, see :ref:`quicktune`
+    InFlight AHRS trim adjust             Auto Trim for Copter, see :ref:`autotrim`
```

Comment:
I wonder if we should unify the names.  Users might not realise that "InFlight AHRS trim" is "Auto Trim" (or "Save Trim"?) and it should probably be consistent with the parameter descriprtion.

Sorry to bring it up so late in the review

Reply:
we would need to change the param metadata....I don't have an issue with calling the aux switch AutoTrim....we can just change it....I can do it if you want

Reply:
Hi @Hwurzburg, yes, let's call it AutoTrim.  I've created a PR here https://github.com/ArduPilot/ardupilot/pull/29352

Reply:
I've actually renamed it to "AHRS AutoTrim" now

Reply:
done